catalog/tests/dags/providers/provider_api_scripts/test_smk.py

```diff
@@ -132,3 +141,27 @@ def test_get_record_data_returns_main_image():
     images = smk.get_record_data(item)
 
     assert len(images) == 1
+
+
+@pytest.mark.parametrize(
+    "property_name",
+    [
+        pytest.param("object_number", id="falsy foreign_landing_url"),
+        pytest.param("rights", id="falsy license_info"),
+    ],
+)
+def test_get_record_data_returns_none_with_falsy_required_property(property_name):
+    item = _get_resource_json("item.json")
+    if property_name == "id":
```

Comment:
What is this check doing?

Reply:
It's left out from copy-pasting. I removed it, and improved tests